api/openapi-spec/swagger.json

```diff
@@ -494,6 +508,12 @@
        "description": "Should the export be exact. Exact export maintains cluster-specific fields like 'Namespace'",
        "name": "exact",
        "in": "query"
+      },
+      {
+       "type": "boolean",
```

Comment:
this one shouldn't be here, I will drop it

Reply:
Actually it is not true, K8S support it here as well, so I will keep it and just change name of PR